hackathon_site/dashboard/frontend/src/pages/Teams/Teams.test.tsx

```diff
@@ -1,13 +1,60 @@
 import React from "react";
 
-import { render } from "testing/utils";
-import { teamsList } from "testing/mockData";
-import Teams from "pages/Teams/Teams";
-
-test("renders teams without crashing", () => {
-    const teamCodes = teamsList.map((team) => team.TeamName);
-    const { getByText } = render(<Teams />);
-    teamCodes.forEach((teamCode) => {
-        expect(getByText("Team " + teamCode)).toBeInTheDocument();
+import { makeMockApiListResponse, render } from "testing/utils";
```

Comment:
Reminder to update all changed files with absolute imports

Reply:
I think I changed them all

Reply:
you should check again (see my other comment)